app/src/main/AndroidManifest.xml

```diff
@@ -36,6 +36,10 @@
     <activity android:name=".ChecklistItemActivity">
     </activity>
     <activity android:name=".AboutActivity"/>
+    <provider
+        android:authorities="com.g11x.checklistapp.provider"
+        android:name=".LocalRepository"
```

Comment:
Authorities are like databases, not tables within the db. It would be rather rare for a single app to have more than one.